README.md

```diff
@@ -1,3 +1,59 @@
-## My test task
+# My test task
+
+---
 
 Here you can find my implementation of OrderBook.
+
+---
```

Comment:
Add build section.
This section will explain, how to build your project

Reply:
Add `Pre-requisites`.
This section will describe the technologies that should be installed on the OS to build/run your program.

Reply:
fix